lms/services/application_instance.py

```diff
@@ -31,6 +31,27 @@ def get(self, consumer_key=None) -> ApplicationInstance:
 
         return application_instance
 
+    def create(  # pylint:disable=too-many-arguments
```

Comment:
Not sure if the `from_url` part is need, it just puts the URL value on the URL column